client/my-sites/sharing/buttons/appearance.jsx

```diff
@@ -99,11 +99,21 @@ class SharingButtonsAppearance extends Component {
 	}
 
 	getReblogLikeOptionsElement() {
-		if ( ( ! this.props.isJetpack || this.props.isLikesModuleActive ) ) {
+		const {
+			isJetpack,
+			isLikesModuleActive,
+			translate
+		} = this.props;
+
+		if ( ( ! isJetpack || isLikesModuleActive ) ) {
 			return (
 				<fieldset className="sharing-buttons__fieldset">
 					<legend className="sharing-buttons__fieldset-heading">
-						{ this.props.translate( 'Reblog & Like', { context: 'Sharing options: Header' } ) }
+						{
+							isJetpack
+								? translate( 'Like', { context: 'Sharing options: Header' } )
```

Comment:
:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `comment`
- `verb: imperative`

 Would it make sense to reuse one of the above?